release/long_running_tests/workloads/serve_failure.py

```diff
@@ -69,13 +69,28 @@ def update_progress(result):
 
 @ray.remote(max_restarts=-1, max_task_retries=-1)
 class RandomKiller:
+
+    # The interval is to make sure controller will be able to broadcast routes
+    # and recover replicas.
+    CONTROLLER_KILL_INTERVAL = 20
```

Comment:
this is a totally orthogonal change, if we include this it's not clear if the other code changes actually solve the problem (the test was running fine before the offending PR).